### DISCUSSION TITLE
Quick newbie question about static blur

### DISCUSSION THREAD

**Discussion by notmde** [aunetx/blur-my-shell#380] [Q&A] [answered]
I always wondered, where does blur my shell extension keeps the static blur? Like the blurred image it renders on top bar, overview and stuff, where does it store it?

**Comment by aunetx** [comment, marked as answer]
Hello, sorry for the very big delay, the extension actually does not store it: it creates a copy of the `Meta.BackgroundActor` (the actor which is used in gnome-shell's code to display the background image), apply a blur effect on it and put it where needed! It is effectively only a variable stored in memory
